server/src/main/java/org/opensearch/http/AbstractHttpServerTransport.java

```diff
@@ -106,6 +121,7 @@ public abstract class AbstractHttpServerTransport extends AbstractLifecycleCompo
     private final Set<HttpServerChannel> httpServerChannels = Collections.newSetFromMap(new ConcurrentHashMap<>());
 
     private final HttpTracer tracer;
+    private final Tracer telemetryTracer;
```

Comment:
```suggestion
    private final Tracer tracer;
```

Reply:
Actually tracer is already defined in line 123 so used different name.

Reply:
> Actually tracer is already defined in line 123 so used different name.

The other one is `httpTracer` right? We could rename that one instead to have naming consistent